### DISCUSSION TITLE
Where to find the latest Docker docker-compose.yaml file?

### DISCUSSION THREAD

**Discussion by DaveMtl** [BlueBrain/nexus#3771] [Q&A]
Greeting everyone,

I'm following the tutorial to install a local cluster using Docker with the [docker-compose.yaml file](https://bluebrainnexus.io/docs/getting-started/running-nexus/index.html#deployment) and I'm having a few problems.

Looking at the file and it's using the old nexus-web 1.5.0 instead of the latest nexus-web 1.8.0. I'm not very familiar with Docker and not yet confident I could properly update it so I'd like to know if there is a new docker-composer.yaml file available that uses all the latest versions of the Nexus platform? Or maybe just some pointers on what to update exactly.

Thanks.

**Comment by imsdu** [comment]
Hello,
Sorry, weare in the process of updating the doc for 1.8 and haven't got through that part yet.
We will update this file and get back to you.
There is other examples of docker-compose files that we use for integration tests here:
https://github.com/BlueBrain/nexus/tree/master/tests/docker
(The fusion part is missing)

**Comment by imsdu** [comment]
Hello @DaveMtl , we deployed [a new version of the snapshot documentation](https://bluebrainnexus.io/snapshot/docs/getting-started/running-nexus/index.html#recommended-docker-swarm) where the docker-compose file has been updated.
Note that you need to download an extra config file to download.

Are you testing on a Apple M1 ?
Because we just set up deploying Docker images for ARM architectures so only the latest image is available for it.
Plus Docker Swarm does not seem to work well for M1 but for local testing, Docker compose should be enough.

Please let us know if it helps.
